addrmgr/addrmanager.go

```diff
@@ -156,8 +221,8 @@ const (
 	minBadDays = 7
 
 	// getAddrMax is the most addresses that we will send in response
-	// to a getAddr (in practice the most addresses we will return from a
-	// call to AddressCache()).
+	// to a getAddr message. Effectively, this defines the most addresses
```

Comment:
Right, updated.

Reply:
I changed the wording on this to be more generic since the actual limit for an addr protocol message differs from this constant, and the number of addresses pushed to the peer are limited to the appropriate value elsewhere (for the current protocol version).